deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore-cluster/base/dhstore/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dhstore
+spec:
+  template:
+    spec:
+      topologySpreadConstraints:
+        - maxSkew: 1
+          topologyKey: topology.kubernetes.io/zone
+          whenUnsatisfiable: ScheduleAnyway
+      containers:
+        - name: dhstore
+          args:
+            - '--storePath=/data'
+            - '--disableWAL'
+          volumeMounts:
+            - name: data
+              mountPath: /data
+          resources:
+            limits:
+              cpu: "6"
+              memory: 60Gi
```

Comment:
Do we want to lower down the spec for dhstores?

Reply:
what instance type do you suggest instead of the current one?

Reply:
I'd probably go down to 2 cpus initially. `dhstore-helga` now runs at 75% CPU utilisation.

![image](https://github.com/ipni/storetheindex/assets/31857042/ca8b1389-1fba-4d93-9baa-96d0f9ba83c2)

WRT RAM - I *think* we can try lowering it down to 10-20Gi, given that instances are going to receive just a 1/10th of the traffic.